addons/website_sale/models/product_product.py

```diff
@@ -103,4 +103,6 @@ def _get_contextual_price_tax_selection(self):
         company_taxes = self.taxes_id.filtered(lambda tax: tax.company_id == self.env.company)
         if line_tax_type == "tax_included" and company_taxes:
             price = company_taxes.compute_all(price, product=self, partner=self.env['res.partner'])['total_included']
+        if line_tax_type == "tax_excluded" and company_taxes.price_include:
```

Comment:
what if there are multiple taxes in `company_taxes` ?

Reply:
Thanks for the review!
I modified it so, whenever there is at least one tax `price_include` and `line_tax_type == "tax_excluded"` , we call the compute_all to get the `tax_exclude` price. I added a this use case in the tests